test/integration/test_migration_runner.py

```diff
@@ -0,0 +1,251 @@
+                            );
+                            """
+        )
+
+        assert rule_name_exists_result.first()[0]
+
+        rule_type_is_filled_result = self.conn.get_records(
+            f"""
+                SELECT COUNT(*)
+                FROM data_quality_test.quality_check_example_table
+                WHERE rule_type='not_null'
+            """
+        )
+
+        assert rule_type_is_filled_result.first()[0] == 3
+
+
+# test of migration from v 0.1.4. to 0.1.5 transaction
+class TestMigrationsResolverTransaction(unittest.TestCase):
+    def setUp(self):
+        """
+        Init a temporary table with some data.
+        """
+
+        sql = [
+            f"DROP SCHEMA IF EXISTS {DATA_QUALITY_SCHEMA} CASCADE;"
+            f"CREATE SCHEMA IF NOT EXISTS {DATA_QUALITY_SCHEMA};",
+            get_quality_table_creation_script_0_1_4(
+                DATA_QUALITY_SCHEMA, DATA_QUALITY_TABLE_1
+            ),
+            get_quality_table_creation_script_0_1_4_invalid(
+                DATA_QUALITY_SCHEMA, DATA_QUALITY_TABLE_2
+            ),
+            f"""
+            INSERT INTO {DATA_QUALITY_SCHEMA}.{DATA_QUALITY_TABLE_1} (attribute, rule_name, rule_description, total_records, failed, median_30_day_failed, failed_percentage, passed, median_30_day_passed, passed_percentage, status, time_filter, task_ts, created_at, id) VALUES ('src', 'not_null', 'True when data is null.', 41136, 0, null, 0, 41136, null, 100, 'valid', null, '2019-11-09 00:00:00.000000', '2019-11-12 12:41:28.391365', 75597);
+            INSERT INTO {DATA_QUALITY_SCHEMA}.{DATA_QUALITY_TABLE_1} (attribute, rule_name, rule_description, total_records, failed, median_30_day_failed, failed_percentage, passed, median_30_day_passed, passed_percentage, status, time_filter, task_ts, created_at, id) VALUES ('dst', 'not_null', 'True when data is null.', 41136, 0, null, 0, 41136, null, 100, 'valid', null, '2019-11-09 00:00:00.000000', '2019-11-12 12:41:28.391365', 75598);
+            INSERT INTO {DATA_QUALITY_SCHEMA}.{DATA_QUALITY_TABLE_1} (attribute, rule_name, rule_description, total_records, failed, median_30_day_failed, failed_percentage, passed, median_30_day_passed, passed_percentage, status, time_filter, task_ts, created_at, id) VALUES ('departure_time', 'not_null', 'True when data is null.', 41136, 0, null, 0, 41136, null, 100, 'valid', null, '2019-11-09 00:00:00.000000', '2019-11-12 12:41:28.391365', 75599);
+            """,
+        ]
+        self.conn = Connector(TEST_DB_URI)
+        for s in sql:
+            self.conn.execute(s)
+
+    def tearDown(self):
+        """
+        Drop all created tables.
+        """
+        self.conn.execute(f"DROP schema data_quality_test CASCADE;")
+        DQBase.metadata.clear()
+
+    def test_migration_to_0_1_5_transaction(self):
+        try:
+            migration.main(
+                ["-u", SQLALCHEMY_URL, "-s", DATA_QUALITY_SCHEMA, "-v", "0.1.5"]
```

Comment:
test are failing now because of the check of the version `if version != contessa.__version__:` at the start of the tests.
Please add global fixture that's gonna override the `contessa.__version__` that will be used in tests. also I'd create fixtures for versions that will be used everywhere in the tests so we avoid this in the future (e.g. actual_version, lower_version, higher_version)